python/tidy/servo_tidy/tidy.py

```diff
@@ -339,10 +339,20 @@ def find_reverse_dependencies(name, content):
         packages_by_name.setdefault(package["name"], []).append((package["version"], source))
 
     for (name, packages) in packages_by_name.iteritems():
-        if name in exceptions or len(packages) <= 1:
+        has_duplicates = len(packages) > 1
+        duplicates_allowed = name in exceptions
+
+        if has_duplicates and duplicates_allowed:
+            continue
+
+        if not (has_duplicates or duplicates_allowed):
```

Comment:
This combined with the line above looks like an exclusive-or to me. Perhaps you can combine the two conditions and use `==` instead?